downloader/artifact_downloader.go

```diff
@@ -147,16 +149,16 @@ func (ad *ConcurrentArtifactDownloader) download(jobs <-chan downloadJob, result
 
 // extractArchive extracts an archive using the tar CLI tool by piping the archive to the command's input.
 func (ad *ConcurrentArtifactDownloader) extractArchive(r io.Reader, targetDir string) error {
-	tarArgs := []string{
-		"-x",      // -x: extract files from an archive: https://www.gnu.org/software/tar/manual/html_node/extract.html#SEC25
-		"-f", "-", // -f "-": reads the archive from standard input: https://www.gnu.org/software/tar/manual/html_node/Device.html#SEC155
-	}
-	cmd := ad.CommandFactory.Create("tar", tarArgs, &command.Opts{
+	cmd := ad.CommandFactory.Create("unzip", []string{}, &command.Opts{
```

Comment:
No need to initialize an empty array here, nil could be passed.

Reply:
Actually I need to pass the zip location here. Updated in a6e05c00863b4b4d060b2e80f43be7915b73ad9d